Copy edits (#132)
websight-io/docs: docs/docs/developers/development/dialogs/include/index.md

## Description

> The Include component allows users to include other components within a parent component. It is typically used to avoid code duplication. When you have a component (e.g., a Select component for heading levels) that is used in multiple places, you can extract it to a common place and include it in multiple locations rather than duplicating it within each one.

Reviewer: ```suggestion
The Include dialog field allows users to include other components within a parent component. It is typically used to avoid code duplication. When you have a dialog field (e.g., a Select dialog field for heading levels) that is used in multiple places, you can extract it to a common place and include it in multiple locations rather than duplicating it within each one.
```

## Customization

> Sometimes it is not enough to include other components and use the default settings provided by the included component. We can easily customize it by creating a child node named `include`. If we do so, we have to set the `sling:resourceSuperType` property. Its value should be the path of the component that we want to include.

Reviewer: ```suggestion
Sometimes it is not enough to include other dialog fields and use the default settings provided by the included dialog field. We can easily customize it by creating a child node named `include`. If we do so, we have to set the `sling:resourceSuperType` property. Its value should be the path of the dialog field that we want to include.
```

> After `sling:resourceSuperType` is set up correctly, we can add properties to the `include` node. Additionally we can create a subtree that is similar to the included component'ss subtree. We can also add or override properties on child nodes.

Reviewer 2: ```suggestion
After `sling:resourceSuperType` is set up correctly, we can add properties to the `include` node. Additionally, we can create a subtree that is similar to the included component's subtree. We can also add or override properties on child nodes.
```
